lib/xcp_service.ml

```diff
@@ -21,6 +21,7 @@ module StringSet = Set.Make(String)
 *)
 let default_service_name = Filename.basename Sys.argv.(0)
 let config_file = ref (Printf.sprintf "/etc/%s.conf" default_service_name)
+let config_dir = ref (Printf.sprintf "/etc/%s.d/conf" default_service_name)
```

Comment:
Thanks -- I think I'll update this to `/etc/%s.conf.d` since that creates the least amount of clashes with existing dirs.